dashboard/app/models/sections/section.rb

```diff
@@ -578,7 +568,29 @@ def update_code_review_expiration(enable_code_review)
   end
   before_validation :strip_emoji_from_name
 
-  public def add_instructor(email, current_user)
+  # Adds an instructor to the section
+  # If the instructor was previously deleted, restore the instructor
+  # If the instructor was invited or inactive, reactive the instructor
+  # If the instructor did not previously exist, create the section instructor relationship
+  # Returns true if the instructor could be added, false otherwise
```

Comment:
Not returning false anymore...

```suggestion
  # Make the instructor active if they had a different status
  # If the instructor did not previously exist, create the section instructor relationship
  # Returns true if successful
```